packages/webapp/src/containers/Finances/OtherExpense/index.js

```diff
@@ -79,9 +79,12 @@ class OtherExpense extends Component {
     let dict = {};
 
     for (let e of expenses) {
+      // Remove the time from the expense date
+      const expenseDate = moment(moment(e.expense_date).format("MM/DD/YYYY"))
+
       if (
-        moment(e.expense_date).utc().isSameOrAfter(moment(startDate)) &&
-        moment(e.expense_date).utc().isSameOrBefore(moment(endDate))
+        moment(expenseDate).utc().isSameOrAfter(moment(startDate)) &&
```

Comment:
Line 83 ensures `expenseDate` is a `moment`, so I think there's no need to call `moment()` on `expenseDate` here, or on the next line.